lib/cve.py

```diff
@@ -28,10 +29,10 @@ def find_cve():
     args = vars(parser.parse_args())
 
     if not os.path.isfile(args['release']):
-        print(f"ERROR: Path to release file {args['release']} doesn't exists")
+        log(f"ERROR: Path to release file {args['release']} doesn't exists")
         file_not_exists = 1
     if not os.path.isfile(args['previousRelease']):
-        print(f"ERROR: Path to previousRelease file {args['previousRelease']} doesn't exists")
+        print.error(f"ERROR: Path to previousRelease file {args['previousRelease']} doesn't exists")
```

Comment:
Does `print.error` exists?

Reply:
not sure. but that is a copy paste issue. I'll fix

Reply:
fixed